README.md

```diff
@@ -103,8 +103,7 @@ system.
     [vc_redist.x64.exe](https://aka.ms/vs/16/release/vc_redist.x64.exe) and [vc_redist.x86.exe](https://aka.ms/vs/16/release/vc_redist.x86.exe)
   * For Linux, the system must have **libgomp.so.1** which can be installed using `apt-get install libgomp1`.
 * GPU builds require CUDA runtime libraries being installed on the system:
-	 * Version: **CUDA 10.0** and **cuDNN 7.3**
-	 * Linux Python packages require **CUDA 10.1** and **cuDNN 7.6** 
```

Comment:
Should we still mention that we need 10.1 for  v0.5 ?

Reply:
I think only the README in the v0.5 branch should talk about 10.1. It can lead to more confusion if we pollute the master branch README with artifacts of previous releases.